src/Rings/mpolyquo-localizations.jl

```diff
@@ -1469,3 +1497,27 @@ end
 function quotient(I::IdealType, J::IdealType) where {IdealType<:MPolyQuoLocalizedIdeal}
   return base_ring(I)(quotient(pre_image_ideal(I), pre_image_ideal(J)))
 end
+
+##############################################################################
+# Further functionality for elements of localized quotients of rings
+##############################################################################
+
+function derivative(f::MPolyQuoLocalizedRingElem, i::Int)
```

Comment:
Please add a test.

Reply:
done

Reply:
Codecove still claims that these lines are not reached by the tests. Could you please double check using e.g. a `@show` statement?

Reply:
Thank you for spotting this Copy&Paste inconsistency in the tests.
Now it should be reached.